internal/lookoutingesterv2/benchmark/benchmark.go

```diff
@@ -194,21 +190,7 @@ func createJobInstructions(jobIds []string, numJobs int, numUserAnnotations int)
 			Annotations:               make(map[string]string),
 		}
 	}
-	createUserAnnotationInstructions := make([]*model.CreateUserAnnotationInstruction, numUserAnnotations)
-	for i := 0; i < numUserAnnotations; i++ {
-		job := createJobInstructions[i%len(createJobInstructions)]
-		k := uuid.NewString()
-		v := uuid.NewString()
-		job.Annotations[k] = v
```

Comment:
I think that we want to keep these lines (otherwise we're just inserting jobs without annotations), but this is not a blocker.

Reply:
ok- yes- let me fix that in a follow up.